### ISSUE TITLE
Last name Proto Site

### ISSUE THREAD

**Issue by PetervanderBurgt** [saproto/saproto#1223]
Dear HYT,

Today I noticed something weird: my name has changed on the website.
Specifically, the first parts of my last name (tussenvoegsels) are not there anymore.
I have looked up some others, and they also seem to have disappeared there.

Kind regards,

Peter

**Comment by PetervanderBurgt** [comment]
Now I have looked into it a bit further, it seemed to have happened only with the people that have been added to the society "Sk8erbois".

**Comment by Jessseee** [comment]
Where in the interface is your name changed? Because I cannot seem to replicate it. Could you send a screenshot?

**Comment by Sebastiaanjai** [comment]
the issue also seems to happen to other members

**Comment by Sebastiaanjai** [comment]
![image](https://user-images.githubusercontent.com/48732717/97433907-840ba100-191e-11eb-95a6-b3a3af042fb7.png)

**Comment by Jessseee** [comment]
Hey @PetervanderBurgt,
We found out what the issue is. The Proto website refreshes your name every time you log in to our website using Surfconext (UT credentials), but it turns out Surfconext or the UT has changed the format of the _last name_ field to no longer contain your affixes. So, every time anyone logs in using Surfconext their affixes disappear from our database. We have made sure this no longer happens, but for the affixes to be easily returned we have to wait for a reply from ICTS.
